VotingApplication/VotingApplication.Web/Api/Controllers/ManageController.cs

```diff
@@ -41,5 +53,94 @@ public ManagePollRequestResponseModel Get(Guid manageId)
         }
 
         #endregion
+
+        #region Put
+
+        public void Put(Guid manageId, ManagePollUpdateRequest updateRequest)
+        {
+
+            #region Input Validation
+
+            if (updateRequest == null)
+            {
+                this.ThrowError(HttpStatusCode.BadRequest);
+            }
+
+            if (updateRequest.Expires && updateRequest.ExpiryDate < DateTime.Now)
+            {
+                ModelState.AddModelError("ExpiryDate", "Invalid or unspecified ExpiryDate");
+            }
+
+            if (!ModelState.IsValid)
+            {
+                this.ThrowError(HttpStatusCode.BadRequest, ModelState);
+            }
+
+            #endregion
+
+            using (var context = _contextFactory.CreateContext())
+            {
+                Poll existingPoll = context.Polls.Where(p => p.ManageId == manageId).Include(p => p.Options).SingleOrDefault();
```

Comment:
This would be a bit clearer if each linq statement was on it's own line, e.g.
Poll existingPoll = context.Polls
                    .Where(p => p.ManageId == manageId)
                    .Include(p => p.Options)
                    .SingleOrDefault();